### DISCUSSION TITLE
sqlalchemy.exc.OperationalError:

### DISCUSSION THREAD

**Discussion by okaven-USGS** [ROBelgium/MSNoise#322] [Q&A] [answered]
Hi folks,
I'm running into the following issues when executing "msnoise compute_dtt":
```
sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1055, "Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'msnoise.jobs.ref' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by")
[SQL: SELECT jobs.ref AS jobs_ref, jobs.day AS jobs_day, jobs.pair AS jobs_pair, jobs.jobtype AS jobs_jobtype, jobs.flag AS jobs_flag, jobs.lastmod AS jobs_lastmod 
FROM jobs 
WHERE jobs.day >= %(day_1)s AND jobs.day <= %(day_2)s AND jobs.jobtype = %(jobtype_1)s AND jobs.lastmod >= %(lastmod_1)s GROUP BY jobs.day ORDER BY jobs.day]
[parameters: {'day_1': datetime.date(2013, 11, 15), 'day_2': datetime.date(2024, 1, 1), 'jobtype_1': 'DTT', 'lastmod_1': datetime.datetime(2024, 1, 31, 14, 30, 6, 910809)}]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```
I suspect I must have made an error in the start/end/ref/ date section. Any clue where the error might be? Thanks in advance for any pointers!

**Comment by ThomasLecocq** [comment]
are you using the 1.6.x version of msnoise, or dev version ?

**Reply by okaven-USGS** [reply]
Hi Thomas, I'm using 1.6.x. Thanks for the reply!

**Reply by ThomasLecocq** [reply, marked as answer]
yah, then it's probably the good old problem of group-by that was present in that version... If your project is not too large, you can run temporarily on sqlite , or go full speed to the unreleased Dev version ...

**Reply by ThomasLecocq** [reply]
if you stay 1.6.x check http://msnoise.org/doc/installation.html?highlight=group#mysql-mariadb-configuration

**Reply by okaven-USGS** [reply]
Thanks for the suggestions, I'll give the dev version a try!

**Reply by ThomasLecocq** [reply]
pleaaaaaaaaaaaaase be indulgent, the doc is not fulllllllllllly updated yet !
